realis_e2e_test.go

```diff
@@ -223,21 +220,53 @@ func TestRealisClient_CreateJob_Thermos(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("TestRealisClient_GetTaskStatus_Thermos", func(t *testing.T) {
```

Comment:
I think you can put all thermos test under a subtest to avoid adding the `_Thermos` suffix, go test will concat the prefix when output `TestRealisClient/Thermos/GetTaskStatus` 

````
t.Run("TestRealisClient, func() {
     t.Run("Thermos", func() {
        t.Run("GetTaskStatus")
     })
})
````

Reply:
Cool, I had no idea they had changed this. I'll get rid of the suffixes.